.gitlab/deploy_containers.yml

```diff
@@ -24,17 +25,19 @@ deploy_containers-a6-on-failure:
     PARENT_PIPELINE_ID: $CI_PIPELINE_ID
     BUCKET_BRANCH: $BUCKET_BRANCH
     FORCE_MANUAL: "true"
+    DDR_WORKFLOW_ID: $DDR_WORKFLOW_ID
```

Comment:
Unless this changed with the new version of gitlab, redefining the same variable name as an already defined variable caused the pipeline to error out when run

Reply:
I'll try to see if this was reported/fixed, but that's indeed the case then I guess we might want to systematically use the invoke task to trigger downstream pipelines

Reply:
It turns out the triggered pipeline already rely on the iknvoke task so there's no need to forward it explicitely from here